Session/Home/App Review/View/AppReviewPromptDialog.swift

```diff
@@ -149,6 +149,15 @@ class AppReviewPromptDialog: UIView {
         secondaryButton.isHidden = prompt?.promptContent.secondaryButtonTitle == nil
         secondaryButton.setTitle(prompt?.promptContent.secondaryButtonTitle, for: .normal)
         secondaryButton.accessibilityIdentifier = prompt?.promptContent.secondaryButtonAccessibilityIdentifier
+        
+        let isButtonsHidden = primaryButton.isHidden && secondaryButton.isHidden
+        
+        buttonStack.layoutMargins = .init(
```

Comment:
Can you please remove the `result.layoutMargins = ...` from the `lazy var buttonStack` creation (since it's always going to be overwritten here having there is redundant)